cpg-language-cxx/src/test/resources/c/dataflow/tls.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
```

Comment:
we could probably keep this, but then we should assert something in the test-case